### PR TITLE
[PW-4652] CC holder name config toggles

### PR DESCRIPTION
**Pull request by acampos1916**
**Description**
When using the Web Component is possible to toggle the holder name field and if it is required. Here a couple of configs are being added to control this feature.

**Tested scenarios**
has_holder_name toggles if the holder name is shown.
holder_name_required toggles if the holder name is required.

**Fixed issue**:  PW-4652

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js [hunk 1]
```diff
@@ -126,6 +126,9 @@ define(
                 self.cardComponent = self.checkoutComponent.create('card', {
                     enableStoreDetails: self.getEnableStoreDetails(),
                     brands: self.getAvailableCardTypeAltCodes(),
+                    hasHolderName: adyenConfiguration.getHasHolderName(),
```
Comment (cyattilakiss) [inline view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js:129]: there is a hasHolderName config already set in the line 80 in this file and also in the adyen-hpp-method.js file. I think those needs to be removed and replaced by this new setting.

Comment (acampos1916) [reply view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js:128]: I think I prefer removing the one from line 80, as it is a card specific config. What do you think?

About the one in adyen-hpp-method.js, is it necessary for hpp methods? If so I can move the config to `Advanced: Checkout Experience` so it applies not only for cards.

Comment (cyattilakiss) [reply view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js:128]: I don't think we actually need it for alternative payment methods
I'm also good with removing it from line 80

Comment (acampos1916) [reply view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js:128]: I just realized that it also affects PMs like bcmc, that use cards in a separate component. I've moved it to checkout experience and adyen_abstract

**Review by AlexandrosMor — APPROVED** [review]

**Review by cyattilakiss — APPROVED** [review]

**Pull request merged**